ingestion/src/metadata/ingestion/source/database/lineage_source.py

```diff
@@ -72,14 +71,6 @@ def get_table_query(self) -> Optional[Iterator[TableQuery]]:
                     for row in rows:
                         query_dict = dict(row)
                         try:
-                            if filter_by_database(
-                                self.source_config.databaseFilterPattern,
-                                self.get_database_name(query_dict),
-                            ) or filter_by_schema(
-                                self.source_config.schemaFilterPattern,
-                                schema_name=self.get_schema_name(query_dict),
-                            ):
-                                continue
                             yield TableQuery(
```

Comment:
Removed this because currently filter patterns in usage and lineage is not being used properly will revisit this for issue  #6829